plugin.video.cbcolympics/changelog.txt

```diff
@@ -0,0 +1,2 @@
+v1.0.0
+- Initial version for 2018 Winter Olympics
```

Comment:
FYI changelogs are no longer used, they were replaced with the <news> parameter in addon.xml.
https://kodi.wiki/view/Addon.xml#.3Cnews.3E

Reply:
you can still keep changelogs for yourself of course

Reply:
@Rechi LGTM

Reply:
@MartijnKaijser True, Just encouraging the use of <news> =)

Reply:
Good to know. I've updated my copy with <news> so the next revision will have it.